person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(name: 'Unknown', age: 0, parent_permission: true)
+    @id = rand(1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def getter_id
+    @id
+  end
+
+  def getter_name
+    @name
+  end
+
+  def getter_age
+    @age
+  end
+
+  def setter_name(name)
+    @name = name
+  end
+
+  def setter_age(age)
+    @age = age
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+
+  def can_use_services?
+    true
+  end
+
+  public :can_use_services?
```

Comment:
- [ ] great job setting up the check for age 💪 however the method `can_use_services?` should return `true` or `false` based on some checks, in this case, you need to check if method `of_age?` returns `true` or `false` or if `@parent_permission` is `true` or `false`.
- [ ] [OPTIONAL] although it works the way you write it here you might want to consider making it look a bit cleaner by doing the following:
```
  def can_use_services?
    true
  end

  private

  def of_age?
    @age >= 18
  end
```
This will work as well because all methods under the `private` keyword will be considered private methods